### PR TITLE
Add scrollable tab bar example

### PR DESCRIPTION
**Pull request by Belobobr**
I have implemented tab bar, that support more than 5 tabs.
This is my first react-native PR, so please be kind.

### CODE REVIEW

File: ScrollableTabBar.js [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+  StyleSheet,
+  ScrollView,
+  TouchableOpacity,
+  Text,
+  Platform,
+} = React;
+
+const TAB_HEIGHT = 50;
+
+var styles = StyleSheet.create({
```
Comment (skv-headless) [inline ScrollableTabBar.js]: move styles to bottom please

File: examples/FacebookTabsExample/FacebookTabsExample.js [hunk 2]
```diff
@@ -59,8 +64,24 @@ var SimpleExample = React.createClass({
...
+          <Text tabLabel='Tab #5'>project</Text>
+        </ScrollableTabView>
+      </View>
+    )
+  }
+});
+
 //module.exports = SimpleExample;
-module.exports = FacebookTabsExample;
+module.exports = ScrollableTabsExample;
```
Comment (skv-headless) [inline examples/FacebookTabsExample/FacebookTabsExample.js]: Add commented string like for SimpleExample

**Comment by skv-headless** [comment]
:+1: super excited

**Comment by gs-akhan** [comment]
Superb !!!! :+1:

**Comment by L-Jovi** [comment]
great, more than 5 tabs is cool!

**Pull request merged**